resources/views/livewire/forum/create-topic-form.blade.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use App\Community\Livewire\Forms\ForumTopicForm;
+
+use function Livewire\Volt\{mount, form, state};
+
+form(ForumTopicForm::class);
+
+// == props
+state(['forum']); // Forum
+
+// == state
+
+// == actions
+$save = function() {
+    $this->form->store();
+};
+
+// == lifecycle
+mount(function() {
+    $this->form->setForum($this->forum);
+});
+?>
+
+<div>
+    <form wire:submit="save">
+        <div class="flex flex-col gap-y-3">
+            <x-base.form.input
+                wire:model="form.title"
+                name="form.title"
+                label="Title"
+                inline
+            />
+
+            <x-base.form.textarea
+                wire:model="form.body"
+                id="input_compose"
+                maxlength="60000"
+                name="form.body"
+                label="Body"
+                rows="22"
+                placeholder="Don't share links to copyrighted ROMs."
+                inline
+                required-silent
+                richText
+            >
+                <x-slot name="formActions">
+                    <button type="submit" class="btn">Submit</button>
```

Comment:
On submit:
> App\Community\Livewire\Forms\ForumTopicForm::store(): Return value must be of type Illuminate\Http\RedirectResponse, Livewire\Features\SupportRedirects\Redirector returned

The topic does get created.

Reply:
Resolved in latest